freebase/scripts/virtuoso

```diff
@@ -151,7 +151,7 @@ def add
     # Virtuoso can't handle prefixes in the second argument, so need to expand.
     tmp = "virtuoso-#{port}-#{i}.ttl"
     log "==== Step A: Converting #{ttl} to #{tmp}..."
-    run "time #{chunkCmd} -indices #{i} | sed -r -e 's/\\tfb:([^\\t]*).$/\\t\\<http:\\/\\/rdf.freebase.com\\/ns\\/\\1\\>./' > #{tmp}"
+    run "time #{chunkCmd} -indices #{i} | sed -r -e 's/\\tfb:([^\\t]*) .$/\\t\\<http:\\/\\/rdf.freebase.com\\/ns\\/\\1\\>./' > #{tmp}"
```

Comment:
You don't always have a space there in ttl files though, so this code wouldn't work if there's no space before the '.'
You could replace [^\\t]* with ([^\\t ]*)[\\t ]*\\.
so that you could handle both cases.